lib/main.dart

```diff
@@ -3,124 +3,114 @@ import 'package:english_words/english_words.dart';
+      return new Column(
+          children: [
+            new IconButton(
+              iconSize: 64.0,
+              onPressed: onPressed,
+              icon: new Icon(icon, color: color)
+            )
+          ],
+      );
+    }
+
+    Widget numPlayersText = new Container(
+      padding: const EdgeInsets.all(32.0),
+      child: new Text(
+        _numPlayers.toString(),
+        style: new TextStyle(
+          fontSize: 32.0,
+        ),
       ),
-      body: _buildSuggestions(),
     );
-  }
 
-  Widget _buildSuggestions() {
-    return new ListView.builder(
-        padding: const EdgeInsets.all(16.0),
-        // The itemBuilder callback is called once per suggested word pairing,
-        // and places each suggestion into a ListTile row.
-        // For even rows, the function adds a ListTile row for the word pairing.
-        // For odd rows, the function adds a Divider widget to visually
-        // separate the entries. Note that the divider may be difficult
-        // to see on smaller devices.
-        itemBuilder: (context, i) {
-          // Add a one-pixel-high divider widget before each row in theListView.
-          if (i.isOdd) return new Divider();
-
-          // The syntax "i ~/ 2" divides i by 2 and returns an integer result.
-          // For example: 1, 2, 3, 4, 5 becomes 0, 1, 1, 2, 2.
-          // This calculates the actual number of word pairings in the ListView,
-          // minus the divider widgets.
-          final index = i ~/ 2;
-          // If you've reached the end of the available word pairings...
-          if (index >= _suggestions.length) {
-            // ...then generate 10 more and add them to the suggestions list.
-            _suggestions.addAll(generateWordPairs().take(10));
-          }
-          return _buildRow(_suggestions[index]);
-        }
+    Widget numPlayersTitle = new Container(
+      padding: const EdgeInsets.all(32.0),
+      child: const Text(
+        "Choose number of players:",
+        style: const TextStyle(fontSize: 16.0),
+      ),
     );
-  }
 
-  Widget _buildRow(WordPair pair) {
+    Widget numPlayers = new Column(
+      children: [
+        numPlayersTitle,
+        new Row(
+          mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+          children: [
+            buildArrowColumn(Icons.arrow_back, () => setState(() {
+              if (_numPlayers > minPlayers) _numPlayers--;
+            })),
+            numPlayersText,
+            buildArrowColumn(Icons.arrow_forward, () => setState(() {
+              if (_numPlayers < maxPlayers) _numPlayers++;
+            }))
+          ],
+        )
+      ],
+    );
 
-    final alreadySaved = _saved.contains(pair);
+    void createRoom() {
+      print('CREATE ROOM');
+    }
 
-    return new ListTile(
-      title: new Text(
-        pair.asPascalCase,
-        style: _biggerFont,
+    Widget createRoomButton = new Container(
+      padding: const EdgeInsets.all(32.0),
```

Comment:
There may be a way of styling things in one go, e.g. set the same paddings everywhere, but we can work it out later :)

Reply:
Yes, I hope so, but I haven't got up to the bit in the docs where it says yet!